MapboxNavigationTests/Support/TokenTestViewController.swift

```diff
@@ -45,10 +45,11 @@ class TokenTestViewController: UIViewController {
         super.viewDidLoad()
         
         DispatchQueue.global().async {
```

Comment:
```suggestion
        DispatchQueue.global().asyncAfter(deadline: .now() + 4) {
```

Reply:
Additional note: I'd prefer explicit comment regarding `asyncAfter(deadline: .now() + 4)` code here.

Reply:
Semaphore approach allows test to be potentially finished faster than 4 seconds, so I thing this variant is better